Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/SharedLogic/Allocator/Jobs/SendSearchRequestToDrivers/Handle/Internal/DriverPool.hs

```diff
@@ -132,7 +134,6 @@ prepareDriverPoolBatch driverPoolCfg searchReq batchNum = withLogTag ("BatchNum-
 
         makeRandomDriverPool batchSize onlyNewDrivers = take batchSize <$> randomizeAndLimitSelection onlyNewDrivers
 
-    takeDriversUsingPoolPercentage :: (MonadFlow m) => ([a], [a]) -> Int -> DriverIntelligentPoolConfig -> m [a]
```

Comment:
why did you remove this?

Reply:
because I need DriverMode param from DriverPoolResult type inside this function, but here in declaration we change the obj type from  DriverPoolWithActualDistResult to **[a]**

Reply:
adding it back as logic will shift before sortingtype check